code/modules/reagents/reagent_containers/borghydro.dm

```diff
@@ -24,6 +24,7 @@ Borg Hypospray
 	var/charge_cost = 50
 	var/charge_timer = 0
 	var/recharge_time = 10 //Time it takes for shots to recharge (in seconds)
+	var/dispensed_temperature = DEFAULT_REAGENT_TEMPERATURE
```

Comment:
This needs to have a doc comment above it.

https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md

Reply:
If I understand this right, I should put 
`///Optional variable to override the temperature add_reagent() will use`
Is that right?

Reply:
Yeah.